src/addons/Pagination.js

```diff
@@ -7,9 +7,11 @@ function renderFraction(h, current, totalCount) {
 
 function renderIndicator(h, index, isCurrent, onClick) {
   return h('li', [
-    h('button', { class: { 'hooper-indicator': true, 'is-active': isCurrent }, on: { click: onClick } }, [
-      h('span', { class: 'hooper-sr-only' }, `item ${index}`)
-    ])
+    h(
+      'button',
+      { class: { 'hooper-indicator': true, 'is-active': isCurrent }, on: { click: onClick }, type: 'button' },
```

Comment:
Good catch, but I'm [pretty sure](https://vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth) the fix should be:

```
{ on: { click: onClick }, attrs: { type: 'button' } }
```

Reply:
Thank you for reviewing my PR! I fixed it at 397f705.